src/BizHawk.Client.Common/rewind/ZwinderBuffer.cs

```diff
@@ -18,6 +18,13 @@ public class ZwinderBuffer : IDisposable
 		*/

```

Comment:
The general problem I have in this file is that ZwinderBuffer is made to do one thing: abstract on top of a ring buffer that's mapped to memory.  With the UseDrive stuff, a bunch of important methods have completely unrelated separate implementations for the two cases.  I don't see that this should all be handled in one file.  If there is commonality, it should be brought out to a base class or base interface or something,